<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SAKOtherAutoResponseItemManager</class>
 <widget class="QWidget" name="SAKOtherAutoResponseItemManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>655</width>
    <height>378</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Auto response settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="5">
    <widget class="QPushButton" name="addItemPushButton">
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QCheckBox" name="forbidAllCheckBox">
     <property name="text">
      <string>Disable all</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="6">
    <widget class="QListWidget" name="listWidget"/>
   </item>
   <item row="1" column="4">
    <widget class="QPushButton" name="deleteItemPushButton">
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="msgLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="outportPushButton">
     <property name="text">
      <string>outport</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="importPushButton">
     <property name="text">
      <string>Import</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
